src/chart/Column.js

```diff
@@ -31,46 +31,62 @@
 
         this._palette = this._palette.switch(this.paletteID());
 
-        var column = this.svgData.selectAll(".columnRect")
+        var insideScale = d3.scale.ordinal();
```

Comment:
rename to "columnScale" and move either this declaration or the initialization code so they are beside each other.